Tests/Common/ZipFileChecker.cs

```diff
@@ -30,28 +27,26 @@ namespace SonarQube.Plugins.Test.Common
     /// </summary>
     public class ZipFileChecker
     {
+        private readonly string unzippedDirectoryPath;
         private readonly TestContext testContext;
+        private readonly ZipArchive archiveFile;
```

Comment:
This is IDisposable, so ZipFileChecker should be too. And be used as `using var`